src/handler/delete.ts

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import { getReasonPhrase, StatusCodes } from 'http-status-codes';
+import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import ExternalApiErrorMessages from '../util/externalApiReferences';
+import HttpResponse from '../util/httpResponse';
+import validAuthorisation from '../util/authorisation';
+import Vehicles from '../util/vehicles';
+import { alreadyRepaired } from '../util/validatorsRecall';
+import logger from '../util/logger';
+
+// eslint-disable-next-line @typescript-eslint/require-await
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  if (!validAuthorisation(event.headers)) {
+    return HttpResponse(StatusCodes.UNAUTHORIZED, getReasonPhrase(StatusCodes.UNAUTHORIZED));
+  }
+  if (!event.queryStringParameters) {
+    return HttpResponse(StatusCodes.BAD_REQUEST, ExternalApiErrorMessages.DvsaAndManufacturerCampaignReferenceMissing);
+  }
+  if (!event.pathParameters) {
+    return HttpResponse(StatusCodes.NOT_FOUND, getReasonPhrase(StatusCodes.NOT_FOUND));
+  }
+  if (!event.pathParameters.vin) {
+    return HttpResponse(StatusCodes.NOT_FOUND, getReasonPhrase(StatusCodes.NOT_FOUND));
+  }
+  if (!event.queryStringParameters.manufacturerCampaignReference && !event.queryStringParameters.dvsaCampaignReference) {
+    return HttpResponse(StatusCodes.BAD_REQUEST, ExternalApiErrorMessages.DvsaAndManufacturerCampaignReferenceMissing);
+  }
+  const { vin } = event.pathParameters;
+  const { manufacturerCampaignReference, dvsaCampaignReference } = event.queryStringParameters;
+  const vehicles = Vehicles;
```

Comment:
why assign to variable with slightly different name?

Reply:
changed this in all 4 functions